apps/costcontrol/js/settings/settings.js

```diff
@@ -128,8 +135,12 @@ var Settings = (function() {
         closeSettings();
       });
 
-      function _setResetTimeToDefault(value, old, key, settings) {
-        var today = new Date();
+      function _setDataPeriodBounds(value, old, key, settings) {
+        var today = InputParser.exportDate(new Date());
```

Comment:
This change was implemented on Bug 1102818 - 'Starting on' field has to be initialized with the current date, following the smartdata specs